src/transport/raw/TCPConfig.h

```diff
@@ -69,7 +69,7 @@ namespace chip {
  * @brief Maximum payload size of a message over a TCP connection
  */
 #ifndef CHIP_CONFIG_MAX_TCP_PAYLOAD_SIZE_BYTES
-#define CHIP_CONFIG_MAX_TCP_PAYLOAD_SIZE_BYTES 1000000
+#define CHIP_CONFIG_MAX_TCP_PAYLOAD_SIZE_BYTES (CHIP_CONFIG_MAX_LARGE_PAYLOAD_SIZE_BYTES)
```

Comment:
I don't understand how this is expected to work.  Can you explain, please?

Reply:
The Large buffer define is in SystemConfig.h